index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>
+        <nav class="navbar">
+          <div class="title"><a>Welcome</a></div>
+          <img src="/images/icon-menu.png" alt="" />
+        </nav>
+      <section class="hero">
+        <h1 class="first">Hey there. I’m Bogdan</h1>
+        <h1 class="second">I’m a software developer</h1>
```

Comment:
- [OPTIONAL] Please for accessibility and SEO, your page should have a single `<h1>` tag, please kindly fix that.